specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -7045,6 +7045,10 @@
       "description": "Google AdWords service linked service properties.",
       "type": "object",
       "properties": {
+        "connectionProperties": {
```

Comment:
These properties were accidentally removed in the [previous PR](https://github.com/Azure/azure-rest-api-specs/pull/25789), which doesn't follow the best practice, so add them back here.

Reply:
generic property bags are not allowed in azure. pl remove them or give them a concrete defintiion.

Reply:
In ADF this is by design to support both primitive type and object type. Btw, as my previous comment mentioned, these properties are not newly added, I remove them unintentional in my previous PR and add them back here to avoid breaking the customers.